numpyro/distributions/continuous.py

```diff
@@ -1188,6 +1188,70 @@ def infer_shapes(
         return batch_shape, event_shape
 
 
+class CAR(Distribution):
+    arg_constraints = {
+        "loc": constraints.real_vector,
+        "alpha": constraints.unit_interval,
```

Comment:
you can use `constraints.open_interval(0, 1)` for this

Reply:
I've read a few more implementations. `alpha` can technically be between -1 and 1, but typically people put a prior on the value lying between 0 (no correlation, iid) and 1 (perfect correlation, although not mathematically possible). But I suppose there might be some cases where people want anticorrelation, so I'll use `constraints.open_interval(-1, 1)` pymc actually allow a vector alpha but I cannot think of a real use case so we'll stick to a single float.